chore: Add Kotlin Consumer and Driver sample apps. (#9)
googlemaps/android-on-demand-rides-deliveries-samples: README.md

1. Driver SDK sample (written in Java) in `/java/driver` directory.
2. Consumer SDK sample (written in Java) in `/java/consumer` directory.
3. Driver SDK sample (written in Kotlin) in `/kotlin/kotlin-driver` directory.
4. Consumer SDK sample (written in Kotlin) in `/kotlin/kotlin-consumer` directory.

> Note: compared to the Java source, the Kotlin source will have

Reviewer: ```suggestion
Note: compared to the Java source, the Kotlin source has
```
